hack/run-functests.sh

```diff
@@ -6,9 +6,8 @@ LATENCY_TEST_RUN=${LATENCY_TEST_RUN:-"false"}
 which ginkgo
 if [ $? -ne 0 ]; then
 	echo "Downloading ginkgo tool"
-	# drop -mod=vendor flags, otherwise the installation will fail
-	# because of the package can not be installed under the vendor directory
-    GOFLAGS='' go install github.com/onsi/ginkgo/ginkgo@v1.16.5
+    go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@latest
```

Comment:
please evaluate if -mod=mod needs to be brought back here.
also I would keep pinning ginkgo not to the latest but to the version we vendor (v2.5.1)

Reply:
with -mod=mod does not work in the ci, it is an error and ginkgo is not found.
Ok to the version

Reply:
> I see a binary files added here : *.test files. What are these files? If these are you own local test files please remove them from this PR.

Sorry trash of running test